client/eventsSDL/InputHandler.h

```diff
@@ -23,6 +23,13 @@ class InputSourceTouch;
 class InputSourceText;
 class InputSourceGameController;
 
+enum class InputModi
```

Comment:
```suggestion
enum class InputModus
```

I think it's more common to use the singular for enum names.

Reply:
thanks :)

Reply:
Turns out 'modus' isn't used in English...

Reply:
Mode?

Reply:
'Mode' is probably better. This is also what we use in code

Reply:
Done :)